onecgiar-pr-client/src/app/shared/services/export-tables.service.ts

```diff
@@ -35,9 +37,11 @@ export class ExportTablesService {
         const workbook = { Sheets: { data: worksheet, 'TOC indicators by result': tocsheet }, SheetNames: ['data', 'TOC indicators by result'] };
         const excelBuffer: any = xlsx.write(workbook, { bookType: 'xlsx', type: 'array' });
         this.saveAsExcelFile(excelBuffer, fileName);
+        callback();
       });
     } catch (error) {
       this.customAlertService.show({ id: 'loginAlert', title: 'Oops!', description: 'Error generating file', status: 'error' });
+      callback();
```

Comment:
@xKeCo as `callback()` is optional, what if when it has to be executed, the function is null/undefined?